### ISSUE TITLE
Improve A11ySnapshotTransformer to match Talkback behaviour on PW

### ISSUE THREAD

**Issue by luizgrp** [google/horologist#1921]
It has a conditional to display only content description when the text semantics is present:

https://github.com/google/horologist/blob/bccd49197af93ae1c863b2ef1888dc1b334aaaf7/roboscreenshots/src/main/java/com/google/android/horologist/screenshots/a11y/A11ySnapshotTransformer.kt#L129-L133

However, as per https://github.com/google/horologist/issues/1920#issuecomment-1873861567, it seems that either compose or talkback are treating nodes with `mergeDescendants = true` as separated nodes.

It looks like the elements fetched by `A11ySnapshotTransformer` should be retrieved with `mergeDescendants = false` to match the behaviour. The side effect is that the screenshots won't "group" the a11y text boxes with the merged descendants.

**Comment by yschimke** [comment]
Partially addressed (against Gustavo's better judgement) in https://github.com/google/horologist/pull/1922/files#diff-d2295c73b71d48e87ce29bdfd15e7c7782a6f1f13e8732d9815034093ec8faa4

**Comment by yschimke** [comment]
This feels beyond our remit, going to close as won't fix. Live with what we have as good enough.
